<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectInfoForm</class>
 <widget class="QWidget" name="ProjectInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
       <item>
        <widget class="QLabel" name="projectName">
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="editDowloadScriptButton">
           <property name="text">
            <string>Edit download script</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="luaDebugButton">
           <property name="text">
            <string>lua debug mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="specButton">
           <property name="text">
            <string>Edit spec file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="showMacrosButton">
           <property name="text">
            <string>Show rpm's macros</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <property name="spacing">
          <number>-1</number>
         </property>
         <item>
          <widget class="QGroupBox" name="localVersionBox">
           <property name="title">
            <string>Local version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="remoteVersionBox">
           <property name="title">
            <string>Remote version</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downloadButton">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buildButton">
           <property name="text">
            <string>Build</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fastBuildButton">
           <property name="text">
            <string>Fast Build</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Build log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <widget class="QPlainTextEdit" name="buildMessages">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_project">
   <property name="text">
    <string>Add project</string>
   </property>
  </action>
  <action name="actionRemove_project">
   <property name="text">
    <string>Remove project</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
